admin/javascript/src/LeftAndMain.EditForm.js

```diff
@@ -121,37 +123,37 @@ $.entwine('ss', function($){
 			// full <form> tag by any ajax updates they won't automatically reapply
 			this.changetracker(this.getChangeTrackerOptions());
 		},
-	
+
 		/**
 		 * Function: confirmUnsavedChanges
-		 * 
+		 *
 		 * Checks the jquery.changetracker plugin status for this form,
 		 * and asks the user for confirmation via a browser dialog if changes are detected.
 		 * Doesn't cancel any unload or form removal events, you'll need to implement this based on the return
 		 * value of this message.
-		 * 
+		 *
 		 * If changes are confirmed for discard, the 'changed' flag is reset.
-		 * 
+		 *
 		 * Returns:
-		 *  (Boolean) FALSE if the user wants to abort with changes present, TRUE if no changes are detected 
+		 *  (Boolean) FALSE if the user wants to abort with changes present, TRUE if no changes are detected
 		 *  or the user wants to discard them.
 		 */
 		confirmUnsavedChanges: function() {
 			this.trigger('beforesubmitform');
-			if(!this.is('.changed')) {
+			if(!this.is('.changed') || this.is('.discardchanges')) {
 				return true;
 			}
 			var confirmed = confirm(i18n._t('LeftAndMain.CONFIRMUNSAVED'));
 			if(confirmed) {
 				// confirm discard changes
-				this.removeClass('changed');
+				this.addClass('discardchanges');
```

Comment:
Can you please document why this class is necessary? I guess it's semantically more correct this way anyway, since form field content is still changed but the user has made the explicit decision to ignore it.